src/common/types.ts

```diff
@@ -47,9 +46,9 @@ export const enum Theme {
 }
 
 export const enum ColorThemeKind {
-	Light = 1,
```

Comment:
Why has this changed? Please revert unless its related to the PR

Reply:
There were some warnings to do with Camel Case.